lib/in_out/saver.py

```diff
@@ -72,15 +72,18 @@ def save_csv(matrix, output_directory, output_file_name):
         Saves the input matrix as a CSV File
 
     Args:
-        nx_graph(str): networkx graph object to be drawn
+        matrix(list):  an array containing data to be saved
         output_drectory(str): location to save graph
-        output_file_name(str): name of the image file to be saved
+        output_file_name(str): name of the csv file to be saved
 
     Returns:
        null
 
     """
     print "Generating", output_file_name + ".csv"
     check_if_dir_exists(output_directory) #create output directory if doesn't exist
-    np.savetxt(output_directory + "/" + output_file_name +".csv", matrix, delimiter=",")
-    
+    file = output_directory + "/" + output_file_name +".csv"
```

Comment:
file is a reserved keyword in python. please don't use that.
Kindly see if this is working correctly for all the instances where this function is being used :
https://github.com/rohangoel96/IRCLogParser/blob/dev/sample.py#L33
https://github.com/rohangoel96/IRCLogParser/blob/dev/sample.py#L55

Compare the previous and latest o/p and add it as a proof in the PR